plugins/hr-resources/src/components/CreateRequest.svelte

```diff
@@ -77,13 +79,33 @@
   function typeSelected (_id: Ref<RequestType>): void {
     type = types.find((p) => p._id === _id)
   }
+
+  function moreThanLimit (
+    employeeRequests: Map<Ref<Staff>, Request[]>,
+    staff: Staff,
+    startDate: Date,
+    endDate: Date,
+    type: RequestType | undefined
+  ): boolean {
+    if (employeeRequests === undefined) return true
+    if (type === undefined) return true
+    const requests = getRequests(employeeRequests, startDate, endDate, staff._id)
+    return requests.length > 0
+  }
+  $: notLimit = moreThanLimit(
+    employeeRequests,
+    staff,
+    new Date(value),
+    new Date(new Date(dueDate).setHours(23, 59, 59, 999)),
+    type
+  )
 </script>
 
 <Card
   label={hr.string.CreateRequest}
   labelProps={{ type: typeLabel }}
   okAction={saveRequest}
-  canSave={value !== undefined}
+  canSave={value !== undefined && !notLimit}
```

Comment:
We need to show a proper message, because it will not be clear why it is not possible to add new.